Libraries/LibWeb/HTML/FormAssociatedElement.h

```diff
@@ -91,10 +91,10 @@ class FormAssociatedElement {
     // https://html.spec.whatwg.org/multipage/forms.html#concept-submit-button
     virtual bool is_submit_button() const { return false; }
 
-    // https://html.spec.whatwg.org/#candidate-for-constraint-validation
+    // https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#candidate-for-constraint-validation
     bool is_candidate_for_constraint_validation() const;
 
-    // https://html.spec.whatwg.org/#concept-fv-valid
+    // https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#concept-fv-valid
```

Comment:
The spec link below this one also needs changing.

Reply:
Sure does! Slipped a "make check-style.py work on Windows" patch in as well.